continuum-auth/src/utils.test.ts

```diff
@@ -0,0 +1,17 @@
+import { Response } from 'node-fetch';
+import { checkStatus } from './utils';
+
+describe('utils', () => {
+    describe('check status', () => {
+        it('asserts on ok status', () => {
+            const status = checkStatus(new Response('', { status: 200 }));
+            expect(status).toBeInstanceOf(Response);
+        });
+        it('throws exception when ok status not found', () => {
+            expect(() => {
+                checkStatus(new Response('', { status: 400 }));
+            }).toThrow(new Error('Bad Request'));
+            // todo - check log written to if exception thrown.
```

Comment:
we dont want to be checking the logs inside a unit test

Reply:
All checking of logs have been removed from the PR.